server/src/main/java/org/elasticsearch/common/logging/NodeAndClusterIdConverter.java

```diff
@@ -49,14 +50,14 @@ public NodeAndClusterIdConverter() {
     }
 
     /**
-     * Updates only once the clusterID and nodeId
+     * Updates only once the clusterID and nodeId.
+     * Note: Should only be called once. Subsequent executions will throw {@link org.apache.lucene.util.SetOnce.AlreadySetException}.
```

Comment:
Nit: Scratch "`Note: Should only be called once.`"?

Reply:
right, the previous line should make it clear that it should only be called once